File: packages/core/src/__tests__/e2e/fixtures/polyglot/poly.go
// E11: unsupported extension (.go). The symbol extractor only walks a known set
// of languages (TS/JS/Python/Dart/Kotlin/...). A .go file should yield zero
// symbols and zero import edges — silently skipped, not an error.

package poly

import "fmt"

func PolyGo(n int) int {
	fmt.Println("poly", n)
	return n + 1
}

type PolyStruct struct {
	ID int
}

File: packages/core/src/__tests__/e2e/fixtures/polyglot/poly.rs
# E11: unsupported extension (.rs). Same expectation as poly.go — zero symbols,
# zero imports. The indexer skips the file silently.

[package]
name = "poly"
version = "0.1.0"

pub fn poly_rust(n: i32) -> i32 {
    n + 2
}

pub struct PolyRs {
    pub id: i32,
}
